core/templates/base-components/base-content.directive.html

```diff
@@ -2,8 +2,9 @@
   <div ng-if="$ctrl.iframed">
     <div tabindex="0" aria-label="Oppia Main Content" id="oppia-main-content" class="protractor-test-main-content"
          ng-cloak>
-      <warning-loader></warning-loader>
-      <div ng-show="!$root.loadingMessage">
+      <loading-message message="$ctrl.loadingMessage"></loading-message>
+      <warning-and-alerts></warning-and-alerts>
+      <div ng-show="!$root.loadingMessage && !$ctrl.loadingMessage">
```

Comment:
Why is this `and`? Shouldn't this be `or`?

Reply:
(I know $root.loadingMessage will go off when you replace all instances)

Reply:
In this case, we have to show the actual content of the webpage and not the loading screen.
The loading screen is to be shown when either `root.loadingMessage` has some value or `ctrl.loadingMessage` has some value. But this ng-show is not for the loading screen. It is for the actual webpage, so the webpage has to be shown when neither the `root.loadingMessage` nor `ctrl.loadingMessage` have some value. It is like
` !(root.loadingMessage or ctrl.loadingMessage)` == `!root.loadingMessage && ctrl.loadingMessage`

Reply:
Ah I see, thanks for the explanation.